collect/collect.go

```diff
@@ -100,17 +209,32 @@ func (c *InventoryCollection) Collect() {
 func (c *InventoryCollection) Upload() error {
 	log.Info().Msg("uploading inventory")
 
-	payload, err := json.Marshal(c.Inventory)
+	var (
+		payload     []byte
+		err         error
+		contentType string = "application/json; charset=UTF-8"
+	)
+
+	payload, err = json.Marshal(c.Inventory)
```

Comment:
Just double checked - `json.Marshal` does serialize to UTF-8 encoding to this should fit the the content type: https://pkg.go.dev/encoding/json#Marshal